recipe/meta.yaml

```diff
@@ -27,7 +23,7 @@ requirements:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
     host:
-        - hdf5 1.8*
```

Comment:
Not sure if libmed works with latest hdf5.

Reply:
If you update to 4.0.0, it does. However, I encountered an error on windows as indicated above.